jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

```diff
@@ -190,7 +190,7 @@ spec:
       scheduled: true
     name: "latest"
 
-## OCP4 Anomaly Detection Image from https://gitlab.cee.redhat.com/AICoE/ocp4
+## OCP4 Anomaly Detection Image from https://github.com/aicoe-aiops/ocp4-anomaly-detection-internal
```

Comment:
Since the repo is internal you should avoid mention it publicly.

Reply:
I don't think we care about this. We reference an internal image repository with the same content as is in this private repo. I don't see a reason why to obfuscate that. If this enables any attack vector in any way, we are doomed already for a long time.

Reply:
It depends. If there is sensitive data then would be good obfuscate or don't mention it publicly.